server/utils/premises/premisesActions.ts

```diff
@@ -21,5 +21,13 @@ export const premisesActions = (user: UserDetails, premises: Premises) => {
     })
   }
 
+  if (user.permissions.includes('cas1_premises_view') && premises.supportsSpaceBookings) {
```

Comment:
Changed to an explicit test for permission - but this permission is already given to users in prod.
The `supportsSpaceBookings` flag however is not set on any premises ATM, and will be set on rollout in February.